spec/assoc.test.sh

```diff
@@ -56,16 +56,22 @@ k2
 []
 ## END
 
-#### Can't initialize assoc array with indexed array
+#### Can initialize assoc array with the "(key value ...)" sequence
 declare -A A=(1 2 3)
 echo status=$?
+argv.py "${!A[@]}"
+argv.py "${A[@]}"
```

Comment:
ditto, could also add declare -p output

Although if it doesn't match bash, it's OK to leave it out, if this test has full coverage

Reply:
I fixed this also in 6c2014b4d.